deepspeech_wrap.h

```diff
@@ -1,20 +1,43 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+    typedef struct MetadataItem {
```

Comment:
I'm wondering if we should have the content of the `struct` pulled from `deepspeech.h` or if you prefer we copy here? There was some include path mess I would have to properly solve for the first solution. The second solution has the advantage of making it easier to read, since contrary to states we don't have just an opaque pointer here.

Reply:
I'm OK with the second solution